os/include/tinyara/logm.h

```diff
@@ -36,6 +36,14 @@ enum logm_loglevels_e {
 	LOGM_OFF  /* Is this needed? */
 };
 
+enum logm_param_type_e {
+	LOGM_BUFSIZE,
+	LOGM_NEW_BUFSIZE,
```

Comment:
We should keep 2 values seperately because resizing buffer isn't operated immediately when the request is entered.
LOGM_BUFSIZE is used in logm_get to get current logm buffer size and LOGM_NEW_BUFSIZE is used in logm_set to set new buffer size.